### PR TITLE
Fix bug 1239021: Correct ALLOWED_ATTRIBUTES for MathML <mn> element

### PR DESCRIPTION
**Pull request by Elchi3**
https://bugzilla.mozilla.org/show_bug.cgi?id=1239021

### CODE REVIEW

File: kuma/wiki/constants.py [hunk 1]
```diff
@@ -106,7 +106,7 @@
 ALLOWED_ATTRIBUTES['mfrac'] += ['bevelled', 'denomalign', 'linethickness',
                                 'numalign']
 ALLOWED_ATTRIBUTES['mi'] += ['dir', 'mathsize', 'mathvariant']
-ALLOWED_ATTRIBUTES['mi'] += ['mathsize', 'mathvariant']
+ALLOWED_ATTRIBUTES['mn'] += ['dir', 'mathsize', 'mathvariant']
```
Comment (willkg) [inline kuma/wiki/constants.py]: Why these three attributes and not `mathbackground` and `mathcolor`?

Comment (Elchi3) [inline kuma/wiki/constants.py]: Those 2 are on all MathMl elements, and get whitelisted in line 81.

**Comment by willkg** [comment]
There's a test for allowed attributes, though I'm not entirely clear on what it does. Seems like it's just verifying that bleach is actually allowing the attributes specified in settings through. Regardless, none of the mathml tags are represented. We probably want to add those or at least write up a bug to add them. That test is in `kuma/wiki/tests/test_content.py` in `AllowedHTMLTests.test_allowed_attributes`.

Other than that, looks good to me.

**Comment by Elchi3** [comment]
Opened https://bugzilla.mozilla.org/show_bug.cgi?id=1241502

**Comment by willkg** [comment]
Given that no one else has chimed in here, I'm going to r+ it. Bug #1241502 covers adding some tests. Beyond that, I think if there are problems here, then it'll become apparent and someone will write up a new bug.

**Pull request merged**